lib/web_ui/test/engine/scene_builder_test.dart

```diff
@@ -16,7 +16,7 @@ void main() {
 
 void testMain() {
```

Comment:
add a regression test for the bug which caused you to have to revert the original PR?